qutebrowser/mainwindow/prompt.py

```diff
@@ -535,6 +542,7 @@ def __init__(self, question, parent=None):
         self._vbox = QVBoxLayout(self)
         self._vbox.setSpacing(15)
         self._key_grid = None
+        self._BASE_COMMANDS = [('prompt-toggle', 'Toggle prompt visibility')]
```

Comment:
This should be at the class level, like `KEY_MODE` above - since it's constant, there is no point in it being saved separately for every instance.

For some more context, see: [Understanding Class and Instance Variables in Python 3 | DigitalOcean](https://www.digitalocean.com/community/tutorials/understanding-class-and-instance-variables-in-python-3)

Reply:
Oops, misread that one.